csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -1302,9 +1302,9 @@ List of relationships (`relationships`) of value type `array`with 1 or more item
       }
     }
 
-The Relationship item is of value type `object` with the three mandatory properties Product Reference (`product_reference`), Relates to Product Reference (`relates_to_product_reference`), and Relationship Type (`relationship_type`) as well as the optional property Full Product Names (`full_product_names`) establishes a link between two existing `full_product_name_t` elements, allowing the document producer to define a combination of two products that form a new `full_product_name` entry.
+The Relationship item is of value type `object` with the four mandatory properties Full Product Name (`full_product_name`), Product Reference (`product_reference`), Relates to Product Reference (`relates_to_product_reference`), and Relationship Type (`relationship_type`) establishes a link between two existing `full_product_name_t` elements, allowing the document producer to define a combination of two products that form a new `full_product_name` entry.
```

Comment:
I'm having a hard time parsing this sentence. Should there be a colon after "four mandatory properties" and the "establishes a link..." part maybe needs a separate sentence?

How about:

```suggestion
The Relationship item is of value type `object` with four mandatory properties: Full Product Name (`full_product_name`), Product Reference (`product_reference`), Relates to Product Reference (`relates_to_product_reference`), and Relationship Type (`relationship_type`). The Relationship item establishes a link between two existing `full_product_name_t` elements, allowing the document producer to define a combination of two products that form a new `full_product_name` entry.
```

If it reads fine for you, feel free to ignore this suggestion ;-)

Reply:
Thanks for the suggestion. I just pushed an additional commit.

Reply:
Excellent, thank you!